fix/fix_am/global_hyblev_fcst_rrfsL65.txt

```diff
@@ -0,0 +1,68 @@
+     2    66
```

Comment:
The `fix` subdir of the regional_workflow was removed awhile back, and it is now stagged locally on each platform under `/path/to/UFS_SRW_App/develop/fix`. I suggest we move this file under the `fix_am` directory there. I'm moving some files from the dev1 branch to the `fix_gsi` directory, so I can help stage this file too.

Reply:
@EdwardSnyder-NOAA  I agree. It makes life easier if such custom fix files are available on each system, so I've removed this file and will rely on fix_am of each system.

Reply:
Added `global_hyblev_fcst_rrfsL65.txt` file to `fix_am` directory on RDHPCs. Still need to add it to odin, stampede, and wcoss2.